games-action/minetest/metadata.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<herd>games</herd>
```

Comment:
I am asking right now at the minetest-dev channel in irc what implifications it might have for users but id rather have it as a useflag that can be enable by users then just enabling it without people knowing that it is there.